py/client/build.gradle

```diff
@@ -99,24 +96,39 @@ def testPyClient = Docker.registerDockerTask(project, 'testPyClient') {
             into 'project/'
         }
     }
+    dockerfile {
+        from('deephaven/cpp-client:local-build')
+        copyFile('project', '/project')
+        workingDir('/project')
+        runCommand '''set -eux; \
+                      pip3 install --upgrade pip; \
+                      pip3 install -r requirements-dev.txt; \
+                      pip3 install .; \
+                      cd /; \
+                      rm -fr /project
+                   '''
+    }
+    parentContainers = [ project.tasks.getByPath(':cpp-client:cppClient') ]
```

Comment:
this file now also needs an evaluationDependsOn :cpp-client

On the existing evaluationDependsOn line: I don't think we need to depend on the python project any more.